src/components/shapes/draw_newshape/attributes.js

```diff
@@ -9,6 +10,75 @@ var shapeLabelTexttemplateVars = require('../label_texttemplate');
 
 module.exports = {
     newshape: {
+        visible: extendFlat({}, basePlotAttributes.visible, {
+            editType: 'none',
```

Comment:
minor point, you can use `overrideAll` if every `editType` is `'none'`

Reply:
Good call. There was a webpack bundling issue doing that which is now fixed by [51a28f9](https://github.com/plotly/plotly.js/pull/6653/commits/51a28f95e7adc7073ed79c0c7fbb439596dbb185).